ux.symfony.com/templates/changelog.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% set meta = {
+    title: 'Changelog',
+    title_suffix: " - Symfony UX",
+    description: 'Symfony UX changelog - New features, bug fixes, performances and security improvments about Symfony Live Components, Twig Components, Autocomplete, Icons...',
```

Comment:
```suggestion
    description: 'Symfony UX changelog - New features, bug fixes, performances and security improvements about Symfony Live Components, Twig Components, Autocomplete, Icons...',
```

Reply:
Also question on the s of twig components 
There are many?

Reply:
Same in paragraph after

Reply:
Thank you @94noni